src/test_utils.rs

```diff
@@ -17,6 +18,18 @@ use crate::{
     BlockId, MAX_TRANSACTION_SIZE, Slot, Transaction, ValidatorId, ValidatorInfo, VotorEvent,
 };
 
+#[derive(Clone, Debug, Serialize, Deserialize)]
+pub struct Ping;
+
+#[derive(Clone, Debug, Serialize, Deserialize)]
+pub struct Pong;
+
+#[derive(Clone, Debug, Serialize, Deserialize)]
+pub enum PingOrPong {
```

Comment:
Do we even still need these at all? Could just use `String` or some primitive type as message type for testing.